.gitignore

```
@@ -2,3 +2,5 @@
 /build*/
 /site/
 bazel-*
+# Bzlmod lockfile
+MODULE.bazel.lock
```

I would guess that the file can only be present in the project root only.
```suggestion
/MODULE.bazel.lock
```

---

done